gradio/themes/base.py

```diff
@@ -136,43 +197,64 @@ def expand_shortcut(shortcut, mode="color", prefix=None):
         self.text_xl = text_size.xl
         self.text_xxl = text_size.xxl
 
+        # Font
+        if not isinstance(font, Iterable):
+            font = [font]
+        self._font = [
+            fontfam if isinstance(fontfam, fonts.Font) else fonts.Font(fontfam)
+            for fontfam in font
+        ]
+        if not isinstance(font_mono, Iterable):
+            font_mono = [font_mono]
+        self._font_mono = [
+            fontfam if isinstance(fontfam, fonts.Font) else fonts.Font(fontfam)
+            for fontfam in font_mono
+        ]
+        self.font = ", ".join(str(font) for font in self._font)
+        self.font_mono = ", ".join(str(font) for font in self._font_mono)
+
+        self._stylesheets = []
+        for font in self._font + self._font_mono:
+            font_stylesheet = font.stylesheet()
+            if font_stylesheet:
+                self._stylesheets.append(font_stylesheet)
+
         self.set()
 
     def set(
         self,
         *,
         # Core Colors
-        color_accent=None,
-        color_accent_soft=None,
-        color_accent_soft_dark=None,
-        color_background_primary=None,
-        color_background_primary_dark=None,
-        color_background_secondary=None,
-        color_background_secondary_dark=None,
-        color_border_accent=None,
-        color_border_accent_dark=None,
-        color_border_primary=None,
-        color_border_primary_dark=None,
-        # Text Colors
-        text_color_code_background=None,
-        text_color_code_background_dark=None,
-        text_color_code_border=None,
-        text_color_link=None,
-        text_color_link_active=None,
-        text_color_link_active_dark=None,
-        text_color_link_dark=None,
-        text_color_link_hover=None,
-        text_color_link_hover_dark=None,
-        text_color_link_visited=None,
-        text_color_link_visited_dark=None,
-        text_color_subdued=None,
-        text_color_subdued_dark=None,
+        background_accent=None,
+        background_accent_soft=None,
+        background_accent_soft_dark=None,
+        background_primary=None,
```

Comment:
I know the omission of `_color` is deliberate since the background can be an image but it's confusing that some attributes have `_color` and some don't. 

I think we should mention in the guide that these can be traditional colors or images (sorry if it's already there, I couldn't find it) to make that clear.

Also, some of these attributes, like `checkbox_background_selected_dark` should have `color` in the name since it can only be a color and not a file right?